src/effects/VST/VSTEffect.h

```diff
@@ -101,6 +101,10 @@ struct VSTEffectSettings
 
 struct VSTEffectWrapper : public VSTEffectLink, public XMLTagHandler
 {
+   VSTEffectWrapper(const PluginPath& path)
```

Comment:
Most one-argument constructors should be `explicit`

Reply:
ok, done